tailwind.config.js

```diff
@@ -150,6 +150,7 @@ module.exports = {
       '48': '12rem',
       '56': '14rem',
       '64': '16rem',
+      '68': '24rem',
```

Comment:
Are the rem values here meant to match pixels so 24rem looks like 68px text? It seems confusing to name these variables numbers that don't reflect their actual values. Why not call them 12, 14, 16.. etc or even better the solution below where its small, medium, large and that way the name isn't coupled to the value which might change over time.

Reply:
I agree the naming is a bit weird but it's from the framework convention isn't it? If it is from the framework we should just leave it as is